client/my-sites/site-monitoring/hooks/use-site-logs-downloader.ts

```diff
@@ -175,7 +175,7 @@ export const useSiteLogsDownloader = ( {
 					{
 						start: startTime,
 						end: endTime,
-						page_size: 10000,
+						page_size: 500,
```

Comment:
Out of curiosity, how does reducing the page size resolve the error? This isn't immediately obvious to me...